ktor-template-core/src/main/kotlin/mobi/waterdog/rest/template/validation/StringFunctions.kt

```diff
@@ -0,0 +1,20 @@
+package mobi.waterdog.rest.template.validation
+
+import org.valiktor.Constraint
+import org.valiktor.Validator
+
+/**
+ * Represents a constraint that validate if the value is a valid phone number
+ */
+object PhoneNumber : Constraint
+
+/**
+ * Validates if the [String] property value is a valid phone number
+ *
+ * @receiver the property to be validated
+ * @return the same receiver property
+ */
+fun <E> Validator<E>.Property<String?>.isPhoneNumber(): Validator<E>.Property<String?> =
```

Comment:
Not sure if this is something that we should add to `core`. This is a very specific helper.